### DISCUSSION TITLE
Canonicalization Methods

### DISCUSSION THREAD

**Discussion by pknowl** [the-human-colossus-foundation/oca-spec#72] [General]
Why does the current approach utilise different canonicalization methods for different parts to calculate SAIDs, when [RFC 8785](https://www.rfc-editor.org/rfc/rfc8785) provides a simpler and more standardised method?

Would the TC be open to simplifying canonicalization by applying RFC 8785 and eliminating the need for complex methods that are unnecessary for human readability and processing?

**Comment by blelump** [comment]
@pknowl please ask the question in the former discussion https://github.com/the-human-colossus-foundation/oca-spec/discussions/69 . It ensures the reasoning about particular topic has full provenance in one place.

**Comment by pknowl** [comment]
Close discussion. It has been integrated into #69.
